### PR TITLE
Добавляет статью Инклюзивные компоненты: список дел

### PR DESCRIPTION
**Pull request by annaindistress**
- Решает Issue #113 

- Исправляет опечатку в слове добавления (строка 80, столбец 207)

### CODE REVIEW

**Comment by pepelsbey** [comment]
@TatianaFokina можешь помочь нам альты из оригинальной статьи перевести, раз уж мы можем в этом формате?

1. A bold heading reading “My Todo List” introduces three todo list items on separate lines. These each have a check box to their left and a bin icon to their right. Underneath them is a text field with a placeholder of “E.g. Adopt an owl” and an Add button. The first todo list item’s checkbox is checked and its text (pick up kids from school) is crossed out to show that it is done.
2. A text input with 'e g adopt an owl' placeholder with an 'add' button to its right.
3. An unlabeled field. The user has forgotten what they are doing and just entered 'erm'.
4. Three examples of the described pattern. The first, fom our todo list, uses an add label for the submit button and is fine. The second example for a search form uses a submit button with search as the label and is also acceptable. The third example has a submit button with just submit as the label and is not advised.
5. Adopt an owl added status message. White text on green background prefixed by a tick icon.
6. The empty state, reading either you've done everything already or there are still things to add to your list. Add your first todo, down pointing arrow. The text input and add button appear below.
7. Three todo list items, each with a dustbin icon to their right for deleting them.
8. A dustbin icon placed next to a cross icon for comparison.
9. Dialog reading ‘are you sure you want to delete this item’ with yes and cancel buttons.
10. The first picture shows the Learn Haskell todo item's bin icon/delete button focused with a blue ring. The second picture shows that todo item removed from the list and focus is nowhere to be seen.

File: content/articles/a-todo-list/index.md [hunk 1]
```diff
@@ -45,20 +45,20 @@ date: "2019-02-18"
 
 Даже если вам кажется, что список задач про сбор вещей менее важен, чем определение того, какие бары лучше посетить, он всё ещё находится на том же уровне с точки зрения принадлежности. Так что у него должен быть тот же уровень заголовка.
 
-Страница, структура которой образована логически вложенными разделами, имеет хорошую визуальную иерархию и помогает пользователям скринридеров составить чёткое представление о ней. Также они используют заголовки в качестве инструментов навигации. Например, в JAWS клавиша 2 переместит вас к следующему разделу, который начинается с заголовка `<h2>`. Клавиша H перемещает к следующему заголовку любого уровня.
+Страница, структура которой образована логически вложенными разделами, имеет хорошую визуальную иерархию и помогает пользователям скринридеров составить чёткое представление о ней. Также они используют заголовки в качестве инструментов навигации. Например, в JAWS клавиша <kbd>2</kbd> переместит вас к следующему разделу, который начинается с заголовка `<h2>`. Клавиша <kbd>H</kbd> перемещает к следующему заголовку любого уровня.
 
-### Примечание: элемент <section>
+### Примечание: элемент `<section>`
```
Comment (A-Ostrovnyy) [inline content/articles/a-todo-list/index.md]: @pepelsbey Почему в статье на Медиуме вы не выделяете теги/методы и пр., при редактировании, а в маркдауне наоборот?

Comment (pepelsbey) [reply content/articles/a-todo-list/index.md]: Потому, что Медиум не умеет код в заголовках. Как и вложенные списки, как и таблицы. Поэтому и хочется свалить оттуда.

Comment (A-Ostrovnyy) [reply content/articles/a-todo-list/index.md]: Понятно, спасибо.

**Comment by TatianaFokina** [comment]
Окей, переведу. А в каком виде это лучше сделать?

**Comment by pepelsbey** [comment]
@TatianaFokina я наверное этот вмёржу, а ты пулреквестом :) Спасибо!

**Comment by pepelsbey** [comment]
@annaindistress спасибо :)

**Pull request merged**